examples/atomic-registry/Makefile

```diff
@@ -21,7 +21,7 @@ install-atomic:
 install-systemd:
 	atomic install $(TEST_IMAGE) $(INSTALLHOST)
 	systemctl enable --now atomic-registry-master.service
-	sleep 10
+	sleep 30
```

Comment:
Yes, polling would be ideal. The issue is I exec docker run which exits immediately. The API may not be up for a bit. We may be able to optimize and automate the setup entirely using systemd. I'd rather not dive into that for this PR.